packages/ui/package.json

```diff
@@ -54,6 +54,7 @@
     "type-fest": "^2.3.4"
   },
   "peerDependencies": {
-    "aws-amplify": "3.x.x || 4.x.x"
```

Comment:
Should we stick with `aws-amplify`, or ask customers to install `@aws-amplify/core` and `@aws-amplify/auth` instead? @dbanksdesign @hvergara

Reply:
If we declare the peer dependencies like this, a customer could still install `aws-amplify` and that would satisfy these peer requirements right?

Reply:
Yes, when I tested this locally in my dev env I didn't specifically download these two dependencies, and already had `aws-amplify` installed. However, maybe legacy imports still needs the `aws-amplify` peer dependency?

Reply:
Maybe `@aws-amplify/ui-react-legacy` still need `aws-amplify` peer dependency since it lives in react package?